docs/ArgoCD Integration.md

```diff
@@ -75,8 +66,12 @@ ARG SOPS_VERSION="3.7.1"
 ARG HELM_SECRETS_VERSION="3.12.0"
 ARG KUBECTL_VERSION="1.22.0"
 
-ENV HELM_SECRETS_HELM_PATH=/usr/local/bin/helm
-ENV HELM_PLUGINS="/home/argocd/.local/share/helm/plugins/"
+# In case, wrapper scripts are used, HELM_SECRETS_HELM_PATH needs to be the path of the real helm binary
```

Comment:
Only a typo: the comma after "`In case`" is not needed: `In case wrapper scripts are used` is ok :-)